pages/_app/index.jsx

```diff
@@ -164,12 +163,12 @@ class App extends NextApp {
     return null
   }
 
-  render() {
+  AppShell = () => {
```

Comment:
It seems to be an antipattern to me. Why do you need this?

Reply:
Actually this commit could be part of https://github.com/oacore/dashboard/pull/263. I need it there because I want to call [store method in Reset component](https://github.com/oacore/dashboard/blob/reset-password/pages/reset/step1.jsx#L20).